test/e2e/project.e2e.js

```diff
@@ -77,8 +77,9 @@ describe('Project Commands', () => {
 			expect(contents.map(stripRoot)).to.eql([
 				'README.md',
 				'project.properties',
-				'src',
-				'src/test-proj.ino'
+				'src/test-proj.cpp',
+				'.gitignore',
+				'.github.workflows/main.yaml'
```

Comment:
I didn't run the e2e tests, but this may be a small typo?

```suggestion
				'.github/workflows/main.yaml'
```